languages/abbreviations/hu.json

```diff
@@ -0,0 +1,14 @@
+{
+    "abbreviations": {},
+    "classifications": {},
+    "directions": {
+        "east": "K",
+        "northeast": "ÉK",
+        "south": "D",
+        "northwest": "ÉNY",
+        "north": "É",
+        "southeast": "DK",
+        "southwest": "DNY",
+        "west": "NY"
```

Comment:
I think these object keys should be in the target language, not English

Reply:
Good catch. The words were untranslated in the iOS navigation SDK project in Transifex, so I copied them from the [Mapbox GL Native](https://www.transifex.com/mapbox/mapbox-gl-native/viewstrings/#hu/foundationstrings-darwin/107223353) project in 3b75660a4cf08ad6cc5323065a633cd6d75bb951.